src/pawns.cpp

```diff
@@ -193,7 +191,7 @@ namespace {
             score -= Backward[opposed][f];
 
         if (connected)
-            score += Connected[opposed][phalanx][more_than_one(supported)][relative_rank(Us, s)];
+            score += Connected[opposed][!!phalanx][more_than_one(supported)][relative_rank(Us, s)];
```

Comment:
!! is traditional SF coding style. I see nothing wrong with that.

At first, it looks strange, but when you get used to the notation it's very compact...